sql/sqlite/inc/TSQLiteRow.h

```diff
@@ -14,12 +14,12 @@
 
 #include "TSQLRow.h"
 
-struct sqlite3_stmt;
+#include <sqlite3.h>
```

Comment:
Isn't a fwd decl enough?

Reply:
No, it has more fancy declaration. Not works this way